### PR TITLE
Get member list

### LINKED ISSUE #3: Get all members of a specific channel
- Method that returns all members of a specific channel as a list of Slack user IDs. This should get the channel that the app is currently installed to.
- https://api.slack.com/methods/conversations.members

### PR DESCRIPTION
**Pull request by onlymattjohnson**
Added two new functions

# get_channel_id
This function takes an API client and channel_name and returns the channel_id needed to get members.

# get_channel_members
This function takes an API client and channel_name and returns the a list of member ids in that channel.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -8,13 +8,48 @@
...
-except SlackApiError as e:
-    # You will get a SlackApiError if "ok" is False
-    assert e.response["ok"] is False
-    assert e.response["error"]  # str like 'invalid_auth', 'channel_not_found'
-    print(f"Got an error: {e.response['error']}")
\ No newline at end of file
+def hello_world():
+    """
+    Test that the API client is working
+    """
```
Comment (cjmabry) [inline main.py:14]: 👍 +1 to docstrings on functions!

File: README.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ CHANNEL_NAME='#channel'
 ```
 4. Run demo app
 ```
-python ./main.py
+pipenv run python ./main.py
```
Comment (cjmabry) [inline README.md:27]: Good catch.

File: main.py [hunk 3]
```diff
@@ -8,13 +8,48 @@
...
+    except SlackApiError as e:
+        # You will get a SlackApiError if "ok" is False
+        assert e.response["ok"] is False
+        assert e.response["error"]  # str like 'invalid_auth', 'channel_not_found'
+        print(f"Got an error: {e.response['error']}")
+
+def get_channel_id(client, channel_name):
+    channel_id = None
+    try:
+        response = client.conversations_list()
```
Comment (cjmabry) [inline main.py]: Curious if this returns private conversations, too. Perhaps if we limit the scoping maybe not?

File: main.py [hunk 4]
```diff
@@ -8,13 +8,48 @@
...
+            channel=channel_name,
+            text="Hello world!")
+        assert response["message"]["text"] == "Hello world!"
+    except SlackApiError as e:
+        # You will get a SlackApiError if "ok" is False
+        assert e.response["ok"] is False
+        assert e.response["error"]  # str like 'invalid_auth', 'channel_not_found'
+        print(f"Got an error: {e.response['error']}")
+
+def get_channel_id(client, channel_name):
```
Comment (cjmabry) [inline main.py]: Might recommend docstring here, or changing function name to be more clear, like `get_channel_id_by_name`

**Review by cjmabry — APPROVED** [review]
Great work! Minor comments, nothing blocking.

File: main.py (hunk 4 above)
Comment (onlymattjohnson) [reply main.py]: Renamed function and added docstirngs

File: main.py (hunk 3 above)
Comment (onlymattjohnson) [reply main.py]: Just pushed an update to limit the pull to public channels.

**Comment by onlymattjohnson** [comment]
@cjmabry I made a few updates, can you take a quick look before I merge?

**Pull request merged**